### PR TITLE
Gather headshots for fellows (#666)

### PR DESCRIPTION
**Pull request by BenWoodford76**
Adding fellowship info to slowly fix issue #666

### CODE REVIEW

File: _people/jess_courtney.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 award: Fellowship
 submitted: false
+careers:
+  - Events Assistant
+links:
+  - type: LinkedIn
+    username: jecourtney
 ---
 
+After Graduating, Jess went on to do a Masters at the University of York. During her two years in York, Jess worked in various historical buildings including York Minster and the York Theatre Royal. Also during this time, she held various positions in the National Student Television Association. She currently works as an Events Assistant for the ss Great Britain Trust.
```
Comment (wjdp) [inline _people/jess_courtney.md]: `ss`?

File: _people/matt_leventhall.md [hunk 2]
```diff
@@ -10,3 +12,4 @@ links:
     href: http://www.mllx.co.uk/
 ---
 
+After Graduating in 2009, Matt began freelancing as a Lighting designer. In 2013 he graduated from the Royal Academy of Dramatic Art, having studied Stage Electrics and Lighting Design. Since then he has gone on to work currently for the National Theatre and in 2014, returned to RADA where he is currently the Assistant Head of Lighting.
```
Comment (wjdp) [inline _people/matt_leventhall.md]: `Lighting designer` :arrow_right: `lighting designer`

File: _people/jess_courtney.md (hunk 1 above)
Comment (BenWoodford76) [inline _people/jess_courtney.md]: As in the ship the ss Great Britain

Comment (wjdp) [inline _people/jess_courtney.md]: Ah, needs to be upper case then doesn't it?

Comment (BenWoodford76) [inline _people/jess_courtney.md]: You would be correct in that yes

**Comment by wjdp** [comment]
Thanks for all of this!

(Pick a better branch name next time 😛)

**Pull request merged**